components/integrations/installed_outgoing_webhooks/installed_outgoing_webhooks.tsx

```diff
@@ -86,29 +96,29 @@ export default class InstalledOutgoingWebhooks extends React.PureComponent {
             this.props.actions.loadOutgoingHooksAndProfilesForTeam(
                 this.props.teamId,
                 Constants.Integrations.START_PAGE_NUM,
-                Constants.Integrations.PAGE_SIZE,
+                parseInt(Constants.Integrations.PAGE_SIZE, 10),
```

Comment:
Can we simply cast this as a number? Just to avoid any potential logic chance.

Reply:
Okay, nevermind then.

Reply:
Picture for reference

<img width="791" alt="Screenshot 2022-07-14 at 10 50 04" src="https://user-images.githubusercontent.com/32863416/178942867-88e41572-9d79-4806-b331-3fac5666670b.png">